launcher/ui/dialogs/ReviewMessageBox.cpp

```diff
@@ -49,9 +50,16 @@ void ReviewMessageBox::appendResource(ResourceInformation&& info)
         itemTop->insertChildren(1, { customPathItem });
 
         itemTop->setIcon(1, QIcon(APPLICATION->getThemedIcon("status-yellow")));
-        itemTop->setToolTip(1, tr("This file will be downloaded to a folder location different from the default, possibly due to its loader requiring it."));
+        itemTop->setToolTip(
+            childIndx++,
+            tr("This file will be downloaded to a folder location different from the default, possibly due to its loader requiring it."));
     }
 
+    auto providerItem = new QTreeWidgetItem(itemTop);
+    providerItem->setText(0, tr("Provider: %1").arg(info.provider));
+
```

Comment:
if adding the `required_by` field then add it's display here.
```suggestion
    
    auto requiredByItem = new QTreeWidgetItem(itemTop);
    requiredByItem->setText(0, tr("Required by: %1").arg(info.required_by));
```

If the `required_by` field uses an `std::option` then of course the check should be made.